tests/camel-itest-osgi/src/test/resources/org/apache/camel/itest/TypeConverter

```diff
@@ -0,0 +1 @@
+org.apache.camel.itest.typeconverter.MyConverter
```

Comment:
Not to be picky, but we should have an ASF license header in these files too. You can copy from any existing similar file name.

Before we do releases we run a RAT check which would detect this problem